docker/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  server:
+    build: ./server
+    image: "iwantbot/server"
+    volumes:
+     - ..:/iwant-bot-mount
+    ports:
+     - "8080:8080"
+  database:
+    build: ./database
```

Comment:
It's not necessary to have separate Dockerfile for postgres as it doesn't add anything to it.